### PR TITLE
Add a guide to backport a commit

### PR DESCRIPTION
**Pull request by fmuyassarov**
Add a guide on how to backport a commit from master to a stable branch.

### CODE REVIEW

File: wow/github-workflow.md [hunk 1]
```diff
@@ -184,3 +184,36 @@ Above command will ensure that the topic branch is removed locally and remotely
 
 It is exactly the same process except that steps 9. 10. and 11. do not happen.
 Instead the code is merged with the internal pull request.
+
+## How to backport
+
+Sometimes you may need to backport a commit (e.g. bug fix) from a master branch
+into the stable release branch. This involves a couple of steps as below.
```
Comment (Xenwar) [inline wow/github-workflow.md:191]: into a stable

Comment (Xenwar) [inline wow/github-workflow.md:191]: as described below.

File: wow/github-workflow.md [hunk 2]
```diff
@@ -184,3 +184,36 @@ Above command will ensure that the topic branch is removed locally and remotely
...
+In this example, we will use `release-0.3` as a stable branch in to which we will backport a
```
Comment (Xenwar) [inline wow/github-workflow.md:192]: as the stable branch

File: wow/github-workflow.md [hunk 3]
```diff
@@ -184,3 +184,36 @@ Above command will ensure that the topic branch is removed locally and remotely
...
+specific commit from the `master` branch.
+
+Create and checkout to a new branch (e.g. `backport_commit_x`) based on the stable branch (e.g. `release-0.3`)
+
+```bash
+git checkout -b backport_commit_x origin/release-0.3
+```
+
+Print out the commits from the `master` branch which aren't in the `release-0.3` branch.
+
```
Comment (Xenwar) [inline wow/github-workflow.md:202]: Add an example of the command and print output here

File: wow/github-workflow.md [hunk 4]
```diff
@@ -184,3 +184,36 @@ Above command will ensure that the topic branch is removed locally and remotely
...
+git cherry-pick -x eface850b
+```
+
+If you don't have conflicts, you can now push it.
+
+```bash
+git push origin backport_commit_x
+```
+
+During the PR submission on the Github UI, make sure that you select the right target branch
```
Comment (Xenwar) [inline wow/github-workflow.md:218]: you have selected

File: wow/github-workflow.md [hunk 5]
```diff
@@ -184,3 +184,36 @@ Above command will ensure that the topic branch is removed locally and remotely
...
+which is in our case would be `release-0.3` branch.
```
Comment (Xenwar) [inline wow/github-workflow.md:219]: , which is in our case is

File: wow/github-workflow.md [hunk 6]
```diff
@@ -184,3 +184,36 @@ Above command will ensure that the topic branch is removed locally and remotely
...
+Here you need to take a copy of a SHA (e.g. `eface850b` for random7) of the specific commit
+that you want to backport into the `release-0.3` branch.
+
+Once you know the SHA of the commit, you can cherry-pick that commit.
+
+```bash
+git cherry-pick -x eface850b
+```
+
+If you don't have conflicts, you can now push it.
```
Comment (furkatgofurov7) [inline wow/github-workflow.md:212]: What happens if we do have conflicts after cherry-picking? Would you mind explaining a bit more if that is the case?

Comment (fmuyassarov) [reply wow/github-workflow.md]: Fixed. Thanks.

**Comment by fmuyassarov** [comment]
@Xenwar , @furkatgofurov7 addressed your comments. Thanks!

**Comment by Xenwar** [comment]
approved once @furkatgofurov7 agrees.

**Review by Xenwar — DISMISSED** [review]

**Comment by furkatgofurov7** [comment]
One more thing, although it is not related to this patch, maybe worth fixing: [this](https://github.com/Nordix/airship-dev-tools/blob/master/wow/github-workflow.md#12-delete-the-branch-when-needed) should be changed IMO, it only deletes remote branch but not local. To delete local branch we need to add something like this to notes maybe:

`git branch -d <topic-branch>` 

Your thoughts @fmuyassarov @Xenwar ?
Otherwise looks good to me!

**Comment by fmuyassarov** [comment]
> git branch -d <topic-branch>

Done.

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**